blog/2022-03-02-os.mdx

```diff
@@ -0,0 +1,13 @@
+---
+slug: os-iasql
+title: IaSQL is now open source
+authors: [depombo, dfellis, aguillenv]
+---
+
+We are excited to announce that IaSQL is now open source. The main repository is under https://github.com/iasql/iasql-engine! As perfectionists, we feel like IaSQL will never be truly ready. However, we believe IaSQL is at the point where it can start to be useful for developers managing infrastructure in the cloud. IaSQL is a SaaS that lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database to represent the definitive state (and status) of your cloud which cannot be achived with a static infrastructure declaration. This means that when you connect an AWS account to an IaSQL instance it automatically backfills the database with your existing cloud resources. IaSQL also makes it possible to version control infrastructure changes, akin to [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code), using any migration system or idempotent stored procedures.
```

Comment:
I'm not completely sure that the idempotent stored procedures are a way to version control your code? I think is another way to develop but it is not related with the version control in my opinion. I mean, in the migration system, you will have a migration that calls the stored procedure.

Reply:
Right, that is an orthogonal way of doing things. You can use stored procs with migrations or just calling them with an interactive query prompt. :)